apps/demos/testing/widgets/datagrid/ToolbarCustomization.test.js

```diff
@@ -0,0 +1,39 @@
+import { createScreenshotsComparer } from 'devextreme-screenshot-comparer';
+import { Selector as $ } from 'testcafe';
+import { runManualTest } from '../../../utils/visual-tests/matrix-test-helper';
+import { testScreenshot } from '../../../utils/visual-tests/helpers/theme-utils';
+
+fixture('DataGrid.ToolbarCustomization')
+  .page('http://localhost:8080/')
+  .before(async (ctx) => {
+    ctx.initialWindowSize = [900, 600];
+  });
+
+runManualTest('DataGrid', 'ToolbarCustomization', ['jQuery', 'React', 'Vue', 'Angular'], (test) => {
+  test.only('ToolbarCustomization', async (t) => {
```

Comment:
```suggestion
  test('ToolbarCustomization', async (t) => {
```
(lint should be failed)